packages/perseus/src/components/sortable.tsx

```diff
@@ -859,22 +846,20 @@ class Sortable extends React.Component<SortableProps, SortableState> {
                         item.endPosition = endPosition;
                     }
                     return item;
-                },
-                this,
-            );
+                })
+                .filter(isDefined);
```

Comment:
Doesn't `.filter(Boolean)` do this?  I thought it used to work in Flow.

Reply:
I doesn't unfortunately. 😞